README.md

```diff
@@ -45,6 +45,43 @@ $stack->push(new CacheMiddleware(), 'cache');
 $client = new Client(['handler' => $stack]);
 ```
 
+# Caching strategies
+
+## Public and private 
```

Comment:
Can you add a note about the respect of the RFC-7234 https://tools.ietf.org/html/rfc7234